src/components/ProductListCard/ProductListCard.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useEffect } from 'react';
+import './ProductListCard.scss';
+import { useNavigate, Link } from 'react-router-dom';
+
+const ProductListCard = props => {
+  const [change, setChange] = useState(true);
+
+  const navigate = useNavigate();
+  const { data } = props;
+  const goToDetail = id => {
+    navigate(`/productdetail/${id}`);
+  };
+
+  const clickLike = (id, isLike) => {
+    console.log(id);
+    console.log(isLike);
+    if (!localStorage.getItem('accessToken')) {
+      return alert('로그인이 필요합니다.');
+    }
+
+    if (props.isLike === false) {
+      fetch('http://10.58.52.96:8000/users/signup', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify({
+          id: props.id,
+          accessToken: '123',
+        }),
+      }).then(() => {
+        setChange(!change);
+      });
+    } else {
+      fetch('http://10.58.52.96:8000/users/signup', {
+        method: 'DELETE',
+        headers: {
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+        body: JSON.stringify({
+          id: props.id,
+          accessToken: '123',
+        }),
+      }).then(() => {
+        setChange(!change);
+      });
+    }
+  };
+
+  return (
+    <div className="productListCard">
+      <div
+        className="productListCardImgBox"
+        onClick={() => {
+          goToDetail(data.id);
+        }}
+      >
+        <img
+          className="picFirst"
+          src={data.productImg[1].url}
+          alt="cardImage"
+        />
+        <img
+          className="picSecond"
+          src={data.productImg[0].url}
+          alt="cardImage"
+        />
+        <div
+          className="picCartBox"
+          onClick={e => {
+            e.stopPropagation();
+            goToDetail('cart');
+          }}
+        >
+          <img
+            className="picCart"
+            src={process.env.PUBLIC_URL + '/images/shopping-cart.png'}
+            alt="cardImage"
+          />
+        </div>
+      </div>
+      <div className="titleBox">
+        <div className="labelBox">
+          {data.isNew ? <div className="newProduct">신제품</div> : null}
+          {data.quantity === 0 ? (
+            <div className="emptyQuantity">일시품절</div>
+          ) : null}
```

Comment:
여기서도 마찬가지로 0이 아닌숫자는 true, 숫자 0은 false를 나타내기 때문에
```suggestion
          {!data.quantity  && (
            <div className="emptyQuantity">일시품절</div>
          ) }
```
이렇게도 표현 할 수 도 있겠지만 남은 수량이 0이라는걸 명확이 표기해 주는것이 코드의 이해도를 높일 수 있기 때문에
```suggestion
          {data.quantity === 0  && (
            <div className="emptyQuantity">일시품절</div>
          ) }
```
라고 표현해주는 쪽이 더 좋아 보입니다